src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1900,6 +1905,33 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  /extensions/resource-owners/page:
+    post:
+      operationId: ResourceOwnersPage
+      summary: Get Resource Owners Page
+      description: |
+        Returns list of all owners of a given resource, ordered by Amount (fungibles)/ Number of items (non fungibles) descending.
+        This endpoint operates only at the current state version, it is not possible to browse historical data.
+        Because of that, it is not possible to offer stable pagination as data constantly changes. Values might change, which might result in gaps or some entries being returned twice.
```

Comment:
I realise I never properly discussed what this endpoint was called on the ticket.

I'm sorry if this feels like a pivot, but can we rename this? As per the name of NG-447, I'd like to call this "Resource Holders" not "Resource Owners", to avoid confusion with the _owner_ of the resources in the engine auth role sense.

To that end, can I ask we rename the various files/names to align? (I guess we can do a CTRL-F for "owner" and fix them in turn).

As for these descriptions:

```suggestion
  /extensions/resource-holders/page:
    post:
      operationId: ResourceHoldersPage
      summary: Resource Holders
      description: |
        A paginated endpoint to discover which global entities hold the most of a given resource.

        More specifically, it returns a page of global entities which hold the given resource, ordered descending by the total fungible balance / total count of non-fungibles stored in vaults in the state tree of that entity (excluding unclaimed royalty balances).

        This endpoint operates only at the **current state version**, it is not possible to browse historical data.

        Because of that, it is not possible to offer stable pagination as data constantly changes. Balances might change between pages being read, which might result in gaps or some entries being returned twice.

        The default maximum page size is X.
```

Can we also spell out what `X` is? As we work through these endpoints, we need to record X so people can work out what their max page size requested is, without getting a 400.